andela/titan-dev-ah-frontend
# #163518664 Highlight

#### What does this PR do?
Highlight and comment
#### Description of Task to be completed?
- Highlight text
- Comment on a highlighted text
- Tests: component, action creators, and reducers.
#### How should this be manually tested?
- clone the repo and check out the following branch `ft-highlight-and-comment-163518664 `
- run `yarn start` and navigate to ```/article/: slug``` and try to select a small section of an article. 
### Related Pivotal tracker ID
#163518664
